connectors/sources/confluence.py

```diff
@@ -1236,32 +1235,26 @@ async def get_docs(self, filtering=None):
                         yield document, None
 
         else:
-            if self.spaces == [WILDCARD]:
-                logger.debug("Including docs from all spaces")
-                configured_spaces_query = "cql=type="
-            else:
-                quoted_spaces = "','".join(self.spaces)
-                logger.debug(
-                    f"Including docs from the following spaces: {quoted_spaces}"
-                )
-                configured_spaces_query = f"cql=space in ('{quoted_spaces}') AND type="
-            await self.fetchers.put(self._space_coro)
-            await self.fetchers.put(
-                partial(
-                    self._page_blog_coro,
-                    f"{configured_spaces_query}{BLOGPOST}&{CONTENT_QUERY}",
-                    BLOGPOST,
+            async for space in self._space_coro():
+                yield space, None
+                self._logger.info(f"Fetching docs from space: {space['key']}")
+                configured_spaces_query = f"cql=space in ('{space['key']}') AND type="
```

Comment:
I think it would be valuable to have this query and the ones below added to debug logs.

Reply:
Added a logger for the CQL in `_page_blog_coro`